src/Algebra/Construct/Sub/Group.agda

```diff
@@ -0,0 +1,40 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Definition of subgroups
+------------------------------------------------------------------------
+
+{-# OPTIONS --safe --cubical-compatible #-}
+
+open import Algebra.Bundles using (Group; RawGroup)
+
+module Algebra.Construct.Sub.Group {c ℓ} (G : Group c ℓ) where
+
+private
+  module G = Group G
+
+open import Algebra.Structures using (IsGroup)
+open import Algebra.Morphism.Structures using (IsGroupMonomorphism)
+import Algebra.Morphism.GroupMonomorphism as GroupMonomorphism
+open import Level using (suc; _⊔_)
+
+record Subgroup c′ ℓ′ : Set (c ⊔ ℓ ⊔ suc (c′ ⊔ ℓ′)) where
+  field
+    Sub : RawGroup c′ ℓ′
```

Comment:
I think the name `Algebra.Construct.Sub.*` is fine, even good. I think the name `Sub` as a field name is not.

`group` ? Because it is a group, it has an independent existing as a group, and acquires a new one when inserted into this record.

Reply:
I'm willing to change the name but I don't like the name `group` - there's already another group lurking in the background and I'm worried they'll be confused